{"author":{"id":"471a882371b38fe965ac5b5f66a4a0c62ced8f63a4bd237bb22a92fe2326cd3e"},"ops":[{"type":3,"author":{"id":"471a882371b38fe965ac5b5f66a4a0c62ced8f63a4bd237bb22a92fe2326cd3e"},"timestamp":1694812165,"metadata":{"github-id":"IC_kwDODbKJfs5moZn2","github-url":"https://github.com/vmiklos/plees-tracker/issues/407#issuecomment-1721866742"},"nonce":"RBS0VUOtu+KAE9UVf2nt4N1BpTQ=","message":"Nice, I can reproduce this, thanks.","files":null},{"type":6,"author":{"id":"471a882371b38fe965ac5b5f66a4a0c62ced8f63a4bd237bb22a92fe2326cd3e"},"timestamp":1694812514,"metadata":{"github-id":"UCE_lALODbKJfs5moZn2zjd5gow"},"nonce":"oOlPSM0ljBkjNa1oAIyJNYaM/j4=","target":"4293c58244ccb6dc38bab3257f126d455655d08fe0a29bdbdc2ff4a7202c479c","message":"Nice, I can reproduce this, thanks.\n\nEdit: possibly all we need is something like this: https://stackoverflow.com/questions/67771324/kotlin-networkonmainthreadexception-error-when-trying-to-run-inetaddress-isreac","files":null},{"type":4,"author":{"id":"471a882371b38fe965ac5b5f66a4a0c62ced8f63a4bd237bb22a92fe2326cd3e"},"timestamp":1694812165,"metadata":{"github-id":"CE_lADODbKJfs5t4FA5zwAAAAJrJFR3"},"nonce":"DP3GwaBfUXTRlVf4+yQIx+/dmW0=","status":2},{"type":4,"author":{"id":"471a882371b38fe965ac5b5f66a4a0c62ced8f63a4bd237bb22a92fe2326cd3e"},"timestamp":1694812172,"metadata":{"github-id":"REE_lADODbKJfs5t4FA5zwAAAAJrJFdM"},"nonce":"x1CW/i6GMEtjvSGtHRhMyHAooVQ=","status":1},{"type":5,"author":{"id":"471a882371b38fe965ac5b5f66a4a0c62ced8f63a4bd237bb22a92fe2326cd3e"},"timestamp":1694812182,"metadata":{"github-id":"UNLE_lADODbKJfs5t4FA5zwAAAAJrJFwi"},"nonce":"Ht+x4ALDMotw3NUEXOhzI2eknGQ=","added":[],"removed":["needinfo"]},{"type":5,"author":{"id":"471a882371b38fe965ac5b5f66a4a0c62ced8f63a4bd237bb22a92fe2326cd3e"},"timestamp":1694812182,"metadata":{"github-id":"LE_lADODbKJfs5t4FA5zwAAAAJrJFwl"},"nonce":"7QQ96MjKxfzsZkAVBquwjqN1EM4=","added":["bug"],"removed":[]}]}